analyzers/tests/SonarAnalyzer.UnitTest/TestCases/StringLiteralShouldNotBeDuplicated.CSharp11.cs

```diff
@@ -22,3 +22,17 @@ public class FooLessThanFiveCharacters
 
     public static readonly string NameReadonly = """foo""";
 }
+
+public class FooNonCompliantStringInterpolation
+{
+    public string NameOne = $"{"BarBar" // Noncompliant {{Define a constant instead of using this literal 'BarBar' 4 times.}}
+        }";
+
+    public string NameTwo = $"{"BarBar" // Secondary
+        }";
```

Comment:
Just to change the formatting a bit between the four assignments:

```suggestion
    public string NameTwo = $"{
            "BarBar" // Secondary
        }";
```